### PR TITLE
[DEVINFRA-806] Updated CHANGELOG and prep for next release

### PR DESCRIPTION
**Pull request by adrian-kong**
# Description

Updates changelog and prep for release 4.5.1

### CODE REVIEW

**Comment by joelynch** [comment]
Hey @adrian-kong you also need to run `make all` again so that all the versions have `-alpha` in the name like in this commit https://github.com/swift-nav/libsbp/commit/9f345eec223d41152f4043993f2c95e39adafbee

**Comment by silverjam** [comment]
What's the plan for releasing rust and java?

File: javascript/sbp/RELEASE-VERSION [hunk 1]
```diff
@@ -1 +1 @@
-4.5.0
\ No newline at end of file
+4.5.1-alpha
```
Comment (silverjam) [inline javascript/sbp/RELEASE-VERSION:1]: `package.json` and `package.lock` seem to be missing, there's a bug in javascript here that it sometimes misses these, you should run `make javascript` and see if it fixes this

**Review by silverjam — APPROVED** [review]

**Comment by silverjam** [comment]
> for the rust, need access to publish

just sent you invites

**Review by notoriaga — APPROVED** [review]

**Pull request merged**